lib/util/merge-confidence/index.spec.ts

```diff
@@ -0,0 +1,149 @@
+import nock from 'nock';
```

Comment:
Use http test utils for nock, otherwise we get memory issues again.

Reply:
I copy and pasted from somewhere. Should they all be migrated by now?

Reply:
Yes, they all should be migrated.

Reply:
FYI
![image](https://user-images.githubusercontent.com/6311784/120839909-d7576900-c569-11eb-8af3-e11f4ecb4cba.png)

Reply:
Can you point me to an example file which is done right?

Reply:
Found it